code/modules/antagonists/heretic/knowledge/void_lore.dm

```diff
@@ -243,25 +273,47 @@
 /datum/heretic_knowledge/ultimate/void_final/proc/on_life(mob/living/source, seconds_per_tick, times_fired)
 	SIGNAL_HANDLER
 
-	for(var/mob/living/carbon/close_carbon in view(5, source))
-		if(IS_HERETIC_OR_MONSTER(close_carbon))
+	for(var/atom/thing_in_range as anything in range(10, source))
+		if(!isatom(thing_in_range)) //Better safe than sorry
 			continue
-		close_carbon.adjust_silence_up_to(2 SECONDS, 20 SECONDS)
+
+		if(iscarbon(thing_in_range))
+			var/mob/living/carbon/close_carbon = thing_in_range
+			if(IS_HERETIC_OR_MONSTER(close_carbon))
+				close_carbon.apply_status_effect(/datum/status_effect/void_conduit)
+				continue
+			close_carbon.adjust_silence_up_to(2 SECONDS, 20 SECONDS)
+			close_carbon.apply_status_effect(/datum/status_effect/void_chill, 1)
+			close_carbon.adjust_eye_blur(rand(0 SECONDS, 2 SECONDS))
+			close_carbon.adjust_bodytemperature(-30 * TEMPERATURE_DAMAGE_COEFFICIENT)
+
+		if(istype(thing_in_range, /obj/machinery/door))
+			var/obj/machinery/door/affected_door = thing_in_range
+			affected_door.take_damage(rand(60, 80))
+
+		if(istype(thing_in_range, /obj/structure/door_assembly))
+			var/obj/structure/door_assembly/affected_assembly = thing_in_range
+			affected_assembly.take_damage(rand(60, 80))
```

Comment:
If you want to save yourself some effort these can go in the same block, since it's the same amount of damage anyways (and it's an atom level proc)

Also has the added benefit of keeping these values matching, to prevent someone from accidentally changing one but not the other unintentionally

```suggestion
		if(istype(thing_in_range, /obj/machinery/door) || istype(thing_in_range, /obj/structure/door_assembly))
			var/obj/affected_door = thing_in_range
			affected_door.take_damage(rand(60, 80))
```